### ISSUE TITLE
Why no install instructions? 

### ISSUE THREAD

**Issue by microfx** [JoeyBallentine/Material-Map-Generator#4]
This is frustrating...

**Comment by joeyballentine** [comment]
This repo assumes you have preexisting knowledge of how to install python and python dependencies. I'm sure you can find plenty of guides for how to that elsewhere. Then just install the dependencies I mention in the readme.

**Comment by microfx** [comment]
Yeah ... doesn't run. AttributeError: module 'cv2' has no attribute 'cv2' it's installed.

**Comment by joeyballentine** [comment]
https://github.com/joeyballentine/Material-Map-Generator/blob/387178b3437847388925fba65dc8ae3899acc3a4/generate.py#L96

Change this line to change the cv2.cv2 to just a single cv2. No clue why that's like that, i should probably remove it

**Comment by microfx** [comment]
thx... now the next problem - probably to do with pathes not being set correctly?

```FileNotFoundError: [Errno 2] No such file or directory: 'utils/models/1x_NormalMapGenerator-CX-Lite_200000_G.pth' ```

**Comment by joeyballentine** [comment]
It probably has to do with what path you're running the script from. Make sure in your terminal you've changed the directory to the root of the repo.

Apologies for the issues, this is kinda old and unmaintained

**Comment by enzyme69** [comment]
I am using Codeformer for dependency
https://github.com/sczhou/CodeFormer

**Comment by joeyballentine** [comment]
That doesn't matter. You must run the script from the tour directory of the repo or it won't be able to find the files. At least, I'm assuming that's what the issue is. Please try this

**Comment by Treata11** [comment]
> That doesn't matter. You must run the script from the tour directory of the repo or it won't be able to find the files. At least, I'm assuming that's what the issue is. Please try this

Hi @joeyballentine

May I ask the precise version of `pt` you've trained the models with?
